containers/unit_tests/TestOffsetView.hpp

```diff
@@ -285,8 +285,8 @@ void test_offsetview_unmanaged_construction() {
 
   {
     // Constructing an OffsetView directly around our preallocated memory
-    Kokkos::Array<int64_t, 1> begins1{{2}};
-    Kokkos::Array<int64_t, 1> ends1{{3}};
+    Kokkos::Array<int64_t, 1> begins1{2};
```

Comment:
We might have done this at some point for ROCM and or OpenMPTarget with IBM XL ..

Reply:
Yeah, `clang` up to version 5 is compaining, see https://godbolt.org/z/fqg7ym.

Reply:
So we can decide for having the warning with `HIP` (since we need the workaround) or with older `clang` versions.